### ISSUE TITLE
Pipeline for fast, single image prediction

### ISSUE THREAD

**Issue by tugot17** [airctic/mantisshrimp#333]
## 📓 New <Tutorial/Example>
**Is this a request for a tutorial or for an example?**  
Example

**What is the task?**
How to do a fast prediction for **single**  image

Let say we start with something like
```python

from mantisshrimp.all import *
from PIL import Image

model = faster_rcnn.model(num_classes=5)

model.load_state_dict(torch.load(weights_path, map_location=lambda storage, loc: storage))
model.eval()

img = Image.open(path)
```

How to add Normalizing transformation and get a prediction without constructing the Parser and Dataset

**Is this example for a specific model?**
FasterRCNN

---
**Don't remove**
Main issue for examples: #39

**Comment by lgvaz** [comment]
We have a snippet for that, still needs to be converted to a tutorial, check it out [here](https://airctic.github.io/mantisshrimp/examples/inference/)

**Comment by tugot17** [comment]
Yeah I works, I haven't seen it. However it might be useful to add Resize tfms to the transformations list

`tfms = tfms.A.Adapter([tfms.A.Resize(384, 384), tfms.A.Normalize()])` So we are sure that the image domain is same

**Comment by lgvaz** [comment]
@tugot17 can this be closed?
